webserver/src/main/java/io/kestra/webserver/controllers/api/ExecutionController.java

```diff
@@ -993,7 +997,7 @@ public HttpResponse<FileMetas> getFileMetadatasFromExecution(
     @ExecuteOn(TaskExecutors.IO)
     @Post(uri = "/{executionId}/restart")
     @Operation(tags = {"Executions"}, summary = "Restart a new execution from an old one")
-    public Execution restartExecution(
+    public EventId restartExecution(
```

Comment:
The `EventId` will be serialized as a string which is not ideal in the REST response. It cannot evolve without introducing breaking changes and it doesn't follow standard API design practices. It's better to define a dedicated DTO for the response, e.g. `record ApiAsyncEvent(String id)` that we can re-use everywhere

Reply:
Good call, this pattern may be reused for the TriggerController also so better to design it carefully